### ISSUE TITLE
用TFOpenAIGPTLMHeadModel.from_pretrained(……,from_pt=True)加载pytorch模型效果不佳

### ISSUE THREAD

**Issue by qiuxia-alone** [thu-coai/CDial-GPT#75]
我从 huggingface 上下载了模型文件，用`OpenAIGPTLMHeadModel.from_pretrained(……)`加载没有问题，但用`TFOpenAIGPTLMHeadModel.from_pretrained(……,from_pt=True)`加载时，碰到如下问题：
```
Some weights of the PyTorch model were not used when initializing the TF 2.0 model TFOpenAIGPTLMHeadModel: ['transformer.h.2.attn.bias', 'transformer.h.5.attn.bias', 'transformer.h.9.attn.bias', 'transformer.h.6.attn.bias', 'transformer.h.0.attn.bias', 'lm_head.weight', 'transformer.h.8.attn.bias', 'transformer.h.3.attn.bias', 'transformer.h.1.attn.bias', 'transformer.h.10.attn.bias', 'transformer.h.4.attn.bias', 'transformer.h.11.attn.bias', 'transformer.h.7.attn.bias']
- This IS expected if you are initializing TFOpenAIGPTLMHeadModel from a PyTorch model trained on another task or with another architecture (e.g. initializing a TFBertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing TFOpenAIGPTLMHeadModel from a PyTorch model that you expect to be exactly identical (e.g. initializing a TFBertForSequenceClassification model from a BertForSequenceClassification model).
All the weights of TFOpenAIGPTLMHeadModel were initialized from the PyTorch model.
If your task is similar to the task the model of the checkpoint was trained on, you can already use TFOpenAIGPTLMHeadModel for predictions without further training.
```
它们生成的`logits`相差很大，而且后者循环生成时遇不到终止`token`，一直到`max_len`结束，请问该如何解决呢？

**Comment by silverriver** [comment]
Some weights of the PyTorch model were not used when initializing the TF 2.0 model TFOpenAIGPTLMHeadModel: ['transformer.h.2.attn.bias', 'transformer.h.5.attn.bias', 'transformer.h.9.attn.bias', 'transformer.h.6.attn.bias', 'transformer.h.0.attn.bias', 'lm_head.weight', 'transformer.h.8.attn.bias', 'transformer.h.3.attn.bias', 'transformer.h.1.attn.bias', 'transformer.h.10.attn.bias', 'transformer.h.4.attn.bias', 'transformer.h.11.attn.bias', 'transformer.h.7.attn.bias']


问题在于有些权重没有加载

我们没有尝试过用TFOpenAIGPTLMHeadModel 加载 huggingface上的模型文件。抱歉不是很清楚原因
